projects/jupyter-server-ydoc/jupyter_server_ydoc/handlers.py

```diff
@@ -110,12 +112,26 @@ async def prepare(self):
                     # it is a transient document (e.g. awareness)
                     self.room = TransientRoom(self._room_id, self.log)
 
-            await self._websocket_server.start_room(self.room)
-            self._websocket_server.add_room(self._room_id, self.room)
+            try:
+                await self._websocket_server.start_room(self.room)
+            except Exception as e:
```

Comment:
Maybe I'm missing something, but didn't we introduced the YRoom [exception handler](https://github.com/jupyter-server/pycrdt-websocket/blob/4e610460f425ef12b8e674df0304fced789aef33/pycrdt_websocket/yroom.py#L224) just for that purpose? It seems we are bypassing all the work we did in `pycrdt-websocket` and instead directly handling the exception here.